apps/src/StudioApp.js

```diff
@@ -509,6 +509,11 @@ StudioApp.prototype.alertIfCompletedWhilePairing = function (config) {
             {msg.pairingNavigatorLink()}
           </a>
         }
+        {config.level.pairingChannelId &&
+          <a href={window.dashboard.project.getPathName('view', config.level.pairingChannelId)}>
```

Comment:
We now import project (rather than relying on window.dashboard), so I think you can just do `project.getPathName`

Reply:
👍